### ISSUE TITLE
real-debrid changes

### ISSUE THREAD

**Issue by BedBug2479** [debridmediamanager/awesome-debrid#10]
torrent option and few other options are removed according to the latest announcement

**Comment by juanmita77** [comment]
Agree. Plex Debrid hooking "0 releases" for every single file I add to wish lists. This need to be solved ASAP, thanks.
